lib/forest_liana/bootstraper.rb

```diff
@@ -44,7 +47,7 @@ def fetch_sti_models(model)
 
     def analyze_model?(model)
       return model && model.table_exists? && !SchemaUtils.habtm?(model) &&
-        SchemaUtils.model_included?(model)
+        SchemaUtils.model_included?(model) && !model.parent.try(:table_name)
```

Comment:
If the current model has a parent and this parent has a table_name, it means the model is already present in Forest. Do not add it twice (it's the case for STI, but it could also be the case for other use cases).